core/conversion/converters/impl/squeeze.cpp

```diff
@@ -25,6 +25,15 @@ auto squeeze_registrations TRTORCH_UNUSED = RegisterNodeConversionPatterns().pat
          dim = selfDim.size() + dim;
        }
 
+       if (selfDim[dim] != 1) {
+         auto identity = ctx->net->addIdentity(*self);

```

Comment:
Instead of the identity layer if we arent going to do anything we could just reassoicate the input ITensor with the output node

Reply:
Yes，i remove the identity layer and reassoicate the input ITensor with the output node when we arent going to do anything. Thanks.